### PR TITLE
Fix mapLine function

### PR DESCRIPTION
**Pull request by yurybotov**
As in the I2CioExpander

### CODE REVIEW

File: src/Octoliner.cpp [hunk 1]
```diff
@@ -146,21 +146,73 @@ int Octoliner::digitalRead(int pin)
...
+    int min = 32767;
+    int max = 0;
+    for (int i = 0; i < 8; i++) {
+        if (binaryLine[i] < min)
+            min = binaryLine[i];
+        if (binaryLine[i] > max)
+            max = binaryLine[i];
+    }
+    // calculate border level
+    int border = min + (max - min) / 2;
```
Comment (nkrkv) [inline src/Octoliner.cpp]: I guess the term “threshold” is more suitable than “border” here.

**Review by nkrkv — APPROVED** [review]
:ok_hand: for merge

A little point below. Might be ignored

**Review by nomad605dis — APPROVED** [review]

**Comment by yurybotov** [comment]
Переименовал border в threshold

**Review by nkrkv — APPROVED** [review]
LGTM

**Review by nomad605dis — APPROVED** [review]

**Pull request merged**